### PR TITLE
Set FT_CLIP cookie on rejection too

### PR DESCRIPTION
**Pull request by keirog**
This’ll be used by the opt-in template script on ft.com, to show the
opt-in link to (logged-in) users who’ve previously rejected clippings
migration.

### CODE REVIEW

File: server/save-article.js [hunk 1]
```diff
@@ -17,16 +17,17 @@ module.exports = (req, res, next) => {
 
 	let redirectUrl;
 
+	// Store the user's permission/rejection for repeat journeys
+	// NEXT=bypass the interstitial in the future
+	// FALCON=still show it, also trigger the opt-in link on ft.com (if logged in)
+	res.cookie('FT_CLIP', (userAccepted) ? 'NEXT': 'FALCON', {
+		maxAge: 31536000000, //1 year
```
Comment (laurieboyes) [inline server/save-article.js]: I've done a bit of googling and I think cookie maxAge is in seconds, so this would set it to 1000 years!

Comment (laurieboyes) [inline server/save-article.js]: Not that it really matters I suppose

**Comment by laurieboyes** [comment]
apart from the above :+1:

File: server/save-article.js (hunk 1 above)
Comment (keirog) [inline server/save-article.js]: Hmm, afaik it's ms: http://expressjs.com/api.html#res.cookie

Comment (laurieboyes) [inline server/save-article.js]: Oh yeah you must be right I can't find what I was looking at now but it must have been about something else (???) Soz! :+1: :+1: :+1: :+1: :+1: :+1:

Comment (keirog) [inline server/save-article.js]: Ha, no worries!

Comment (keirog) [inline server/save-article.js]: Ah, the [`max-age` Cache-Control directive is in seconds](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9) but Express' maxAge property is ms

**Pull request merged**